fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/query/expression/DebugExpNodeVisitor.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.database.utils.query.expression;
```

Comment:
This class  appears ideal for src/test/java
if other classes need  too use it they can refer to it in the test dependency

Reply:
Can't move. It's used in other non-test classes